src/test/java/com/wikia/webdriver/common/core/AlertHandler.java

```diff
@@ -1,12 +1,12 @@
 package com.wikia.webdriver.common.core;
 
+import com.wikia.webdriver.common.core.configuration.Configuration;
+
 import org.openqa.selenium.NoAlertPresentException;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-import com.wikia.webdriver.common.core.configuration.Configuration;
-
 public class AlertHandler {
   private AlertHandler() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)